EKF/sensor_range_finder.hpp

```diff
@@ -67,11 +67,11 @@ class SensorRangeFinder : public Sensor
 	// TODO: move the ring buffer here
 	rangeSample* getSampleAddress() { return &_sample; }
 
-	void setSensorTilt(float new_tilt)
+	void setTiltOffset(float new_tilt_offset)
 	{
-		if (fabsf(_tilt - new_tilt) > FLT_EPSILON) {
-			_sin_tilt_rng = sinf(new_tilt);
-			_cos_tilt_rng = cosf(new_tilt);
+		if (fabsf(_tilt_offset_rad - new_tilt_offset) > FLT_EPSILON) {
```

Comment:
Can you explain me what the `_tilt_offset_rad` variable is used for? IMO the variable name does not match the current functionality.

Reply:
Good catch, the purpose is to avoid re-computing the cos and sin function every time, but since I never store the value, it's obviously not working

Reply:
updated, you can re-approve